desktop/utils/domain/auctions/urls.js

```diff
@@ -0,0 +1,19 @@
+const sd = require('sharify').data
+
+/** Get the live auction url with /login appended when a user is present
+ *  Uses sd.PREDICTION_URL or falls back to process.env in case sd is not fully initialized yet
+ * @param {String} id - auction id/slug
+ * @param {Object} [options]
+ * @param {Boolean} [options.isLoggedIn] - whether there is a user present
+ * @return {String} live auction url
+ */
+export const liveAuctionUrl = (id, options = {}) => {
+  const liveAuctionRoot = sd.PREDICTION_URL || process.env.PREDICTION_URL
```

Comment:
There are cases where a redirect happens seemingly before the sharify data is loaded. I went to use the config module (linked in a comment above) and it said to use `process.env`... Likewise that is why i use process.env in the routes.js redirect test. I'd be open to doing this another way but I'm just getting conflicting info on which approach to take at this point.